code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -2,63 +2,92 @@
 /obj/flock_structure
```

Comment:
```suggestion
ABSTRACT_TYPE(/obj/flock_structure)
/obj/flock_structure
```

should probably be abstract; this might cause the checks to fail but that's a good thing